lib/ios/RNNCommandsHandler.m

```diff
@@ -55,23 +55,23 @@ - (instancetype)initWithControllerFactory:(RNNControllerFactory*)controllerFacto
 
 - (void)setRoot:(NSDictionary*)layout commandId:(NSString*)commandId completion:(RNNTransitionCompletionBlock)completion {
 	[self assertReady];
-
+	
 	if (@available(iOS 9, *)) {
-	    if(_controllerFactory.defaultOptions.layout.direction.hasValue) {
-	        if ([_controllerFactory.defaultOptions.layout.direction.get isEqualToString:@"rtl"]) {
-                [[RCTI18nUtil sharedInstance] allowRTL:YES];
-                [[RCTI18nUtil sharedInstance] forceRTL:YES];
-                [[UIView appearance] setSemanticContentAttribute:UISemanticContentAttributeForceRightToLeft];
-                [[UINavigationBar appearance] setSemanticContentAttribute:UISemanticContentAttributeForceRightToLeft];
-            } else {
-                [[RCTI18nUtil sharedInstance] allowRTL:NO];
-                [[RCTI18nUtil sharedInstance] forceRTL:NO];
-                [[UIView appearance] setSemanticContentAttribute:UISemanticContentAttributeForceLeftToRight];
-                [[UINavigationBar appearance] setSemanticContentAttribute:UISemanticContentAttributeForceLeftToRight];
-            }
-	    }
+		if(_controllerFactory.defaultOptions.layout.direction.hasValue) {
```

Comment:
Checking in any of this indent / whitespace changes in these files could cause some unnecessary conflicts in the future if we try to pull in latest from `react-native-navigation`

Reply:
good point. i'll address it